hms-app/src/components/HackathonSelectDropdown.tsx

```diff
@@ -45,13 +45,15 @@ export default function HackathonSelectDropdown(setHackathonID: Props) {
     loadHackathons()
   }, [])
 
-  setHackathonID.setHackathonID(selectedHackweek)
+  useEffect(() => {
+    setHackathonID.setHackathonID(selectedHackweek)
```

Comment:
the props could be like `{ setHackathonID }` so you don't have to call setHackathonID.setHackathonID()